shared/src/business/useCases/docketEntry/fileCourtIssuedDocketEntryInteractor.js

```diff
@@ -31,128 +31,160 @@ exports.fileCourtIssuedDocketEntryInteractor = async (
     throw new UnauthorizedError('Unauthorized');
   }
 
-  const { docketEntryId, docketNumber } = documentMeta;
+  const { docketEntryId, docketNumbers, subjectDocketNumber } = documentMeta;
```

Comment:
Nit: Consider checking the feature flag here as well, and pruning down this list of consolidated docket numbers to just the lead/subject case if false